contracts/messagebuscontract/MessageBus.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity >=0.7.0 <0.9.0;
+
+import "./IMessageBus.sol";
+import "./Structs.sol";
+
+contract MessageBus is IMessageBus {
+
+    function messageFee() virtual internal returns (uint256) { return 0; }
+
+    event LogMessagePublished(address sender, uint64 sequence, uint32 nonce, uint32 topic, bytes payload, uint8 consistencyLevel);
+
+    mapping (bytes32 => uint256) messageFinalityTimestamps;
+    mapping ( address => mapping ( uint32 => Structs.CrossChainMessage[] ) ) messages; 
```

Comment:
Currently no. This will be reiterated in the next PR that brings the bridge as I will be restructuring the smart contracts there to support the optimism contracts as close as possible https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/contracts/L1/messaging/L1CrossDomainMessenger.sol